Dockerfile.wasi-builder

```diff
@@ -38,7 +38,7 @@ RUN git clone --recurse-submodules https://github.com/kateinoigakukun/wasi-vfs
 RUN cd wasi-vfs && \
       git checkout ${WASI_VFS_VERSION} && \
       cargo build --release -p wasi-vfs-cli && \
-      cargo build --release --target=wasm32-wasi
```

Comment:
The main reason is that it's documented that way and I skipped that part of the docs 🙈. I think the idea is that the generated archive (what this command is generating) is standalone in the sense that is linked against a wasi application, so that there's no need to build for the wasi system on this artifact as well.